### PR TITLE
New Personal Library Example

### PR DESCRIPTION
**Pull request by nhcarrigan**
Using the [new boilerplate](https://github.com/freeCodeCamp/boilerplate-project-library/pull/5), I have put together an example project which passes [the proposed /learn tests](https://github.com/freeCodeCamp/freeCodeCamp/pull/39642).

I left the styling untouched, and tried to keep the code similar to what a camper at this point in the curriculum should be able to read and understand.

I have a [Repl.it instance available](https://personal-library.nicholascarriga.repl.co/) for testing against the /learn PR.

/cc @SaintPeter @RandellDawson

### CODE REVIEW

File: personal-library/routes/api.js
Comment (RandellDawson) [inline personal-library/routes/api.js:18]: To handle the two deprecation warnings:
```suggestion
Mongoose.connect(MONGODB_CONNECTION_STRING,  { useNewUrlParser: true, useUnifiedTopology: true }, (err) => {
```

**Review by RandellDawson — CHANGES_REQUESTED** [review]
The example project changes correctly passes all the `/learn` changes on the [corresponding learn PR](https://github.com/freeCodeCamp/freeCodeCamp/pull/39642). 

Just have two change suggestions.

File: personal-library/routes/api.js
Comment (nhcarrigan) [reply personal-library/routes/api.js:18]: Whoops! Meant to go back and add this.

**Review by RandellDawson — APPROVED** [review]

**Comment by RandellDawson** [comment]
@nhcarrigan These changes look good to me know. Once this approve by another mod, we will still need to wait until the changes on the corresponding `/learn` PR have hit production before we can merge it.

Thanks for putting this together so quickly!

**Comment by nhcarrigan** [comment]
Glad to help!

**Review by moT01 — APPROVED** [review]
LGTM 🎉 Should be good to go once the learn PR hits production 😄

**Comment by RandellDawson** [comment]
The `/learn` changes have hit production, so I am merging this now.  I will restart the project in a few minutes.

**Pull request merged**
